unstructured/ingest/v2/README2.md

```diff
@@ -0,0 +1,311 @@
+    LocalUploaderConfig,
+)
+from unstructured.ingest.v2.processes.partitioner import PartitionerConfig
+
+if __name__ == "__main__":
+    Pipeline.from_configs(
+        context=ProcessorConfig(
+            verbose=True,
+            work_dir="local-working-dir",
+            reprocess=True,
+            re_download=True,
+        ),
+        source_connection_config=LocalConnectionConfig(),
+        indexer_config=LocalIndexerConfig(input_path="example-docs/fake-text.txt"),
+        downloader_config=LocalDownloaderConfig(),
+        partitioner_config=PartitionerConfig(),
+        uploader_config=LocalUploaderConfig(output_dir="local-working-dir/output"),
+    ).run()
+```
+You can run this with `python local.py` (Adjust the `input_path` and `output_dir` as appropriate.)
+
+The result would be a partitioned `fake-text.txt.json` file in the `local-output` directory.
+
+
+
+Notice that the pipeline runs the following:
+
+* context - The ProcessorConfig runs the pipeline. The arguments are related to the overall pipeline. We added some optional args to make development easier.
+* source_connection - Takes arguments needed to connect to the source. Local files don't need anything here. Other connectors will.
+* indexer - Takes the files in the `input_path` and creates .json files that point the downloader step to the right files 
+* downloader - This does the actual downloading of the raw files (for non-blob files it may do something different like create a .txt file for every row in a source table)
+* partitioner - Partitions the downloaded file provided it is a partionable file type.
+* chunker/embedder - *Not represented here* but often needed to prepare files for upload to a vector database.
+* stager - *Not represented here* but is often used to prepare partitioned files for upload.
+* uploader - Uploads the blob-like files to the `output_dir`.
+
+
+If you look at the folders/files in `local-working-dir` you will see the files that the pipeline creates as it runs.
+
+```
+local-working-dir
+- index
+  - a4a1035d57ed.json
+- output
+  - fake-text.txt.json
+- partition
+  - 36caa9b04378.json
+```
+
+(Note that the index and partition file names are deterministic and based on the BLABLABLA) In the case of the local source connector, it won't *download* files because they are already local. But for other source connectors there will be a `download` folder. Also note that the final file is named based on the original file with a `.json` extension since it has been partitioned. Not all output files will be named the same as the input file. An example is a table as a source file, the output will be based on a hash of BLABLABLA.
+
+You can see the source/destination connector file that it runs here:
+
+https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/v2/processes/connectors/local.py
+
+If you look through the file you will notice these Classes (actually @dataclasses because BLABLABLA) and functions
```

Comment:
@rbiseck3 can you leave a comment on the BLABLABLA

Reply:
```suggestion
If you look through the file you will notice these interfaces
```